pkl-core/src/main/java/org/pkl/core/module/ModuleKeys.java

```diff
@@ -771,4 +777,56 @@ public boolean hasElement(SecurityManager securityManager, URI elementUri)
       return projectResolver.getResolvedDependenciesForPackage(packageUri, dependencyMetadata);
     }
   }
+
+  public static class MessageTransport implements ModuleKey {
```

Comment:
```suggestion
  public static class External implements ModuleKey {
```

Reply:
Hmm, I had renamed this to align with `ResourceReaders.MessageTransport` since, like that class, this one is used by both external readers and client readers. I think I'd prefer to keep this name and preserve the symmetry. Thoughts?

Reply:
Ooh I see.

Maybe `ResourceReaders.ExternalResolver`?

And then call the other one `ResourceReaders.ExternalProcess`?

And then apply the same names on the module side?